tests/functional/kata-deploy/gha-run.sh

```diff
@@ -13,11 +13,34 @@ kata_deploy_dir="$(dirname "$(readlink -f "$0")")"
 source "${kata_deploy_dir}/../../gha-run-k8s-common.sh"
 
 function run_tests() {
+	cleanup_runtimeclasses || true
+
 	pushd "${kata_deploy_dir}"
 	bash run-kata-deploy-tests.sh
 	popd
 }
 
+function cleanup_runtimeclasses() {
+	# Cleanup any runtime class that was left behind in the cluster, in
+	# case of a test failure, apart from the default one that comes from
+	# AKS
+	for rc in `kubectl get runtimeclass -o name | grep -v "kata-mshv-vm-isolation" | sed 's|runtimeclass.node.k8s.io/||'`
```

Comment:
Do we also want to exclude the `runc` runtime class here? I think it's added by AKS, same as `kata-mshv-vm-isolation`.

Reply:
`runc` is the default one, so that's never actually explicitly shown.